monitoring/README.md

```diff
@@ -120,17 +142,11 @@ Operator and aggregates other helm charts such as Grafana and the Prometheus
 Node Exporter. Links to the charts and default values are included in the
 `user-values-prom-operator.yaml` file.
 
-**Note 1:** If you are using a cloud provider, you must use ingress, rather than
-NodePorts. Use the samples in the [samples/ingress](/samples/ingress)
-area of this repository to set up either host-based or path-based ingress.
+#### TLS Support
 
-**Note 2:** Although the Grafana helm chart supports a value to set the initial
-admin password, you must use the `GRAFANA_ADMIN_PASSWORD` value in
-`$USER_DIR/monitoring/user.env` to set this value. If not specified, the
-initial Grafana admin password will be randomly generated and logged during the
-initial deployment of the monitoring components.
+If you use ingress, connections between the user (or ingress object) and the monitoring components use TLS by default. The [TLS Monitoring sample](/samples/tls/monitoring) contains information about specifying the `TLS_ENABLE` environment variable to use TLS for in-cluster communications between the components and to use TLS for connections between the user and the monitoring components when using NodePorts. If you use ingress and also use TLS for communication between monitoring components, you must also manually populate Kubernetes secrets as listed in the sample. 
```

Comment:
"If you use ingress, connections between the user (or ingress object) and the monitoring components use TLS by default." - This is not true. Use of TLS in the ingress objects is independent of the use of in-cluster TLS between ingress and the backend pods.

When using NodePorts and not ingress, user's connect directly to the backend pods and TLS_ENABLE determines whether they use HTTP or HTTPS.

Reply:
Sentence removed

Reply:
Last sentence changed to "If you use ingress and also use TLS for communication between monitoring components, you do not have to specify the `TLS_ENABLE` environment variable, but you must manually populate Kubernetes secrets as listed in the sample."

Reply:
"If you use ingress and also use TLS for communication between monitoring components, you do not have to specify the TLS_ENABLE environment variable" is not true. "If you only use TLS (HTTPS) for ingress, you do not have to specify the environment variable TLS_ENABLE=true, but you must manually populate Kubernetes ingress secrets as in the TLS sample." (link 'TLS sample' to samples/tls).

Reply:
Changed as noted